projects/pt1/e2e_testing/main.py

```diff
@@ -192,21 +180,21 @@ def main():
         config = FxImporterTestConfig(LinalgOnTensorsTosaBackend(), "tosa")
         xfail_set = FX_IMPORTER_TOSA_XFAIL_SET
         crashing_set = set()
-    elif args.config == "torchdynamo":
-        # TODO: Enanble runtime verification and extend crashing set.
-        config = TorchDynamoTestConfig(
-            RefBackendLinalgOnTensorsBackend(generate_runtime_verification=False)
-        )
-        xfail_set = TORCHDYNAMO_XFAIL_SET
-        crashing_set = TORCHDYNAMO_CRASHING_SET
-    elif args.config == "onnx":
```

Comment:
The support for ONNX comes from the `python/torch_mlir/extras/onnx_importer.py`. I understand that it does not depend on jit_ir_importer. Maybe we should keep this CI for the ONNX backend.

Reply:
have you read this https://github.com/llvm/torch-mlir/pull/3654#issuecomment-2303188109. The onnx also depend on jit_ir_importer this is why I delete it.